frontend/src/app/components/Header.js

```diff
@@ -152,6 +153,9 @@ export default class Header extends React.Component {
 }
 
 Header.propTypes = {
+  uninstallAddon: React.PropTypes.func.isRequired,
+  sendToGA: React.PropTypes.func.isRequired,
+  openWindow: React.PropTypes.func.isRequired,
   hasAddon: React.PropTypes.bool,
   forceHideSettings: React.PropTypes.bool
```

Comment:
`forceHideSettings` looks orphaned now; nothing sets it. I'll add a todo to delete it.